### ISSUE TITLE
Is it possible to get the table data instead of relationships through the api?

### ISSUE THREAD

**Issue by watfop** [saltcorn/saltcorn#674]
I have a few tables with keys that link to other tables. When I pull the tables through the API I get the keys linking to other tables so:
```
{
  success:[
    {
      value1:'from this table',
      value2: 4 //link to other table
    }
  ]
}
```
Can I get the actual value instead of the key?

**Comment by mywebmasteruk** [comment]
any updates on this? thanks

**Comment by glutamate** [comment]
implemented in #2006. use `dereference=key_field_name` in the query string, example in tests:

```
/api/patients/?dereference=favbook
```

(`favbook` is a field on `patients` table, key to the Books table)
